src/sources/http_server.rs

```diff
@@ -88,10 +88,16 @@ pub struct SimpleHttpConfig {
 
     /// A list of HTTP headers to include in the log event.
     ///
+    /// Accepts wildcard (`*`) character for headers matching a specified pattern.
```

Comment:
```suggestion
    /// Accepts the wildcard (`*`) character for headers matching a specified pattern.
```

Reply:
addressed in https://github.com/vectordotdev/vector/pull/18922/commits/4c319da8cbdb8199be9ddeabc8ff6f253134899f